openmc/mgxs_library.py

```diff
@@ -1084,10 +1297,9 @@ class MGXSLibrary(object):
     """
 
     def __init__(self, energy_groups):
-        self._xsdatas = []
-        self._energy_groups = energy_groups
+        self.energy_groups = energy_groups
         self._inverse_velocities = None
```

Comment:
I tend to think of the inverse velocities just like any other cross section. They show up in the transport equation with space dependence and the MGXS module tallies them with space dependence. Is there a reason to they couldn't just be treated like any other xs?